test/artefacts/am-i-getting-minimum-wage/past_payment/2008-10-01/apprentice_under_19.txt

```diff
@@ -1,6 +1 @@
-
-
-There was no National Minimum Wage rate for apprentices before 1 October 2010. Your employer doesn't owe you any past payments.
-
-
-
```

Comment:
404 Error appear here..  for 2008-10-01 related pages

I guess this is unintentional and probably generated by mistake.. kindly take note of this.